upb/reflection/file_def.c

```diff
@@ -181,7 +181,7 @@ bool upb_FileDef_Resolves(const upb_FileDef* f, const char* path) {
   return false;
 }
 
-static char* strviewdup(upb_DefBuilder* ctx, upb_StringView view) {
+static char* _strviewdup(upb_DefBuilder* ctx, upb_StringView view) {
```

Comment:
I renamed this because there is the same name function in `upb/util/def_to_proto.c`. 
https://github.com/protocolbuffers/protobuf/blob/c8ad77475634f77468cd237dd10bcff978a2372a/upb/util/def_to_proto.c#L57

The amalgamation merges `upb` files into one file. So, it causes a compile error when the functions have the same name.